src/pages/beacon.hbs

```diff
@@ -216,5 +216,16 @@
             }
             clock();
         </script>
+        <script src="https://unpkg.com/nosleep.js@0.12.0/dist/NoSleep.min.js" integrity="sha384-lp0XiAMtqqqjyVLBYjAcDQzxc2XyGj3sGESoiWbccHdpXjNUPUjWrq5GMqaGHGp9" crossorigin="anonymous"></script>
```

Comment:
Ideally, we'd be installing `nosleep.js` with `yarn` and bundling it into this page, but I didn't know how to get that working with Handlebars, so I went the easy way out for now. The `integrity` attribute should ensure that if `unpkg.com` suddenly starts serving a different file for whatever reason, then they browser won't actually run it.